### PR TITLE
resource_control: add runaway metrics

### PR DESCRIPTION
**Pull request by CabinfeverB**
### What problem does this PR solve?

Issue Number: ref #43691

Problem Summary:

### What is changed and how it works?
1. add metrics for runaway manager.

2. format the resource-control grafana json.

### Check List

Tests 

- [ ] Unit test
- [ ] Integration test
- [x] Manual test (add detailed scripts or steps below)
image

- [ ] No need to test
  > - [ ] I checked and no code files have been changed.
  >   

Side effects

- [ ] Performance regression: Consumes more CPU
- [ ] Performance regression: Consumes more Memory
- [ ] Breaking backward compatibility

Documentation

- [ ] Affects user behaviors
- [ ] Contains syntax changes
- [ ] Contains variable changes
- [ ] Contains experimental features
- [ ] Changes MySQL compatibility

### Release note

Please refer to [Release Notes Language Style Guide](https://pingcap.github.io/tidb-dev-guide/contribute-to-tidb/release-notes-style-guide.html) to write a quality release note.

```release-note
None
```

### CODE REVIEW

**Comment by CabinfeverB** [comment]
cc @nolouch @glorv

**Review by glorv — APPROVED** [review]

File: domain/resourcegroup/runaway.go [hunk 1]
```diff
@@ -252,6 +253,7 @@ func (rm *RunawayManager) DeriveChecker(resourceGroupName, originalSQL, sqlDiges
 	if group.RunawaySettings == nil && rm.activeGroup[resourceGroupName] == 0 {
 		return nil
 	}
+	metrics.RunawayCheckerCounter.WithLabelValues(resourceGroupName, "hit", "").Inc()
```
Comment (nolouch) [inline domain/resourcegroup/runaway.go:256]: It's better to remove `WithLabelValues`. in most cases, the group name is not changed.

**Comment by CabinfeverB** [comment]
PTAL @nolouch

**Review by nolouch — APPROVED** [review]

**Pull request merged**